### ISSUE TITLE
Dicom file has nested dicomimage under pixeldata tag but when trying to render I get pixeldata tag missing exception

### ISSUE THREAD

**Issue by Muttleskovic** [fo-dicom/fo-dicom#1530]
**Describe the bug**
Trows exception when trying to render image from attachment
Other Dicom viewer's opens the file with no issue
I didn't find any example or info on how to open this type of DICOM Image

**To Reproduce**
DicomImage.RenderImage().AsClonedBitmap()

**Screenshots or test DICOM files**
[29206315.zip](https://github.com/fo-dicom/fo-dicom/files/10921334/29206315.zip)
![image](https://user-images.githubusercontent.com/20122218/223735023-7940356f-4585-41be-ac71-1c9d6932852d.png)
![image](https://user-images.githubusercontent.com/20122218/223735496-1e8fe4c2-70b9-408c-a2bb-7010d3487dd6.png)

**Environment**
Fellow Oak DICOM version: (5.0.3)
OS: (e.g. Windows 10 x64)
Platform: (e.g. .NET Framework 4.7)

Thank you very much

**Comment by mrbean-bremen** [comment]
The image is not valid DICOM, for example `dcmdump` shows:
```
> dcmdump 29206315
W: DcmSequenceOfItems: Reached end of stream before the end of sequence PixelData (7fe0,0010)
E: dcmdump: Sequence Delimitation Item missing: reading file: 29206315
```
and `dcmdump` is usually good at gracefully handling invalid DICOM.
Some viewers seem to be able to compensate for the problem (WEASIS is not able to open it, but MicroDicom is - haven't tried others), so there must be a way to detect and repair the problem, though it is certainly not a bug if the image cannot be displayed.

**Comment by Muttleskovic** [comment]
Indeed it's can't be called a bug, didn't had any issues until this weird DICOM.
microdicom doesn't even shows those 2 tags, got it from radiant. If I try to get it the sequence trows tag not in dataset exception. Radiant sees it as an OB.
Is there a method to get that item nested in pixeldata?

**Comment by Muttleskovic** [comment]
Just found another MR machine that saved DICOM in the same style, looks like "Siemens MAGNETOM" does this.
The sad part is that everything I've tried i get the same exception : Tag : Pixeldata not found in dataset.
[32272542.zip](https://github.com/fo-dicom/fo-dicom/files/10940845/32272542.zip)

**Comment by Muttleskovic** [comment]
I've did some research on web and it seems this is a well know issues with siemens DICOM. RadiAnt had this issue in the past but they manage to mitigate it some how. Sadly I don't have the skill and knowledge to contribute and find a work-around with fo-dicom.
Today I've tested with DCMTK, but they seems to have the same issue "DcmSequenceOfItems: Reached end of stream before the end of sequence PixelData (7fe0,0010)"
At this moment will have to wait for future fo-dicom versions, maybe someone will find a way.
Until then thank all contributing, <3 fo-dicom
